docs/setup/access.asciidoc

```diff
@@ -1,22 +1,33 @@
 [[access]]
-== Accessing Kibana
+== Access {kib}
 
-Kibana is a web application that you access through port 5601. All you need to do is point your web browser at the
```

Comment:
Isn't this information still important for users who are running Kibana on prem?

Reply:
Most definitely, but this page isn't for on-prem. This information should go on an on-prep-specific page.